frontend/packages/dev-console/src/components/pipelineruns/detail-page-tabs/PipelineRunDetails.tsx

```diff
@@ -19,16 +19,45 @@ export const PipelineRunDetails: React.FC<PipelineRunDetailsProps> = ({ obj: pip
           <ResourceSummary resource={pipelineRun} />
         </div>
         <div className="col-sm-6">
+          <SectionHeading text="Pipeline" />
           <dl>
-            <dt>Pipeline</dt>
             <dd>
-              <ResourceLink
-                kind={referenceForModel(PipelineModel)}
-                name={pipelineRun.spec.pipelineRef.name}
-                namespace={pipelineRun.metadata.namespace}
-              />
+              <React.Fragment key={pipelineRun.spec.pipelineRef.name}>
+                <dt>Name: {pipelineRun.spec.pipelineRef.name}</dt>
+                <ResourceLink
+                  kind={referenceForModel(PipelineModel)}
+                  name={pipelineRun.spec.pipelineRef.name}
+                  namespace={pipelineRun.metadata.namespace}
+                  inline
+                />
+              </React.Fragment>
             </dd>
           </dl>
```

Comment:
Not sure why you did this change, please undo.

Reply:
I have changed this because the design is not matching with the Task/Cluster task in pipeline page.

Cluster task link

![image](https://user-images.githubusercontent.com/9964343/77701597-b5b13800-6fdc-11ea-958f-e5c7f7fa2b3d.png)
Pipeline link
![image](https://user-images.githubusercontent.com/9964343/77701567-a205d180-6fdc-11ea-9721-b37187a955f2.png)

Reply:
Designs said otherwise. You are adding resources, not modifying the Pipeline link. The Pipeline link matched designs because there can only be one and was mimicking the design on the left side. If UX wants to chime in and request the change, that's fine, just don't go doing it on your own. If this was a bigger PR, we could have missed this and altered the design.

Reply:
Ah got it. Only Pipeline resource links needs to be matching the cluster task design. Will update it

Reply:
updated it